deploy/addons/dashboard-svc.yaml

```diff
@@ -21,6 +21,8 @@ metadata:
   labels:
     app: kubernetes-dashboard
     kubernetes.io/cluster-service: "true"
+    minikube/addons: dashboard
```

Comment:
could you put this under kubernetes.io/minikube-addons and kubernetes.io/minikube-addons-endpoint